webapp/app/landing/landingState.js

```diff
@@ -4,12 +4,21 @@ export const stateKey = 'landing'
 
 const keys = {
   feed: 'feed',
+  overview: 'overview',
+  users: 'users',
+  countryMessageBoardUnreadMessages: 'countryMessageBoardUnreadMessages',
 }
 
 const getState = R.prop(stateKey)
 
 // === READ
 export const getFeed = R.pipe(getState, R.propOr(null, keys.feed))
+const getOverview = R.pipe(getState, R.prop(keys.overview))
+export const getOverviewUsers = R.pipe(getOverview, R.prop(keys.users))
+export const getCountryMessageBoardUnreadMessages = R.pipe(
+  getOverview,
+  R.propOr(0, keys.countryMessageBoardUnreadMessages)
+)
```

Comment:
why users and countryMessageBoardUnreadMessages are sub properties of overview? remove overview and move them under landing store

Reply:
This is from old code